frontend/test/metabase-visual/models/editor.cy.spec.js

```diff
@@ -20,6 +20,7 @@ describe("visual tests > models > editor", () => {
         cy.visit(`/model/${MODEL_ID}/query`);
         cy.wait("@cardQuery");
         cy.findByText(/Doing science/).should("not.exist");
+        cy.wait(100); // waits for the colums widths calculation
```

Comment:
@nemanjaglumac do you happen to know a way we can avoid `wait` usages here?

Reply:
I did not find a convenient way to wait for table finishes resizing which can be async 😕